docs/ios/xcsync.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Xcode Sync (xcsync) for .NET MAUI
```

Comment:
this is the title that shows up on the side tab..since this already under the .NET MAUI docs umbrella, is this specification redundant? alternative title ideas welcome :)

Reply:
Yes, it's redundant. I'd go with something like "Xcode sync" for now.